### PR TITLE
Cleanup framework angular dependencies

### PR DESCRIPTION
**Pull request by kroeder**
## What I did

I noticed that the framework/angular/package.json needed a cleanup.

For checking missing or obsolete dependencies, I used `npx depcheck code/frameworks/angular`

## How to test

- [ ] Is this testable with Jest or Chromatic screenshots?
- [ ] Does this need a new example in the kitchen sink apps?
- [ ] Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Comment by ndelangen** [comment]
I think right now.. the dependency on react can't be removed... I think?

**Comment by kroeder** [comment]
Who is responsible for providing a react-version in this mono-repo? 

Nothing references react in frameworks/angular. I assumed that react is provided by a different package that needs react installed.

I can undo that, though. Any way to test if this change is ok or not?

**Comment by kroeder** [comment]
The test runner figured out that something is missing :) I get the tests green first, lets discuss further changes aftwards

**Comment by kroeder** [comment]
@ndelangen Since I'm not sure whether we need @types for react as dep or not, I re-added them.

For the rest, here's a quick summary

- **fork-ts-checker-webpack-plugin** and **ts-loader** can be removed due to https://github.com/storybookjs/storybook/pull/19368/files#diff-07dabeeca96ca88626539ed6cff49c2ca71caf34a51d068596571d83ed3441bbL1
- **autoprefixer** can be removed due to https://github.com/storybookjs/storybook/pull/19368/files#diff-0009efa19beb6daf6560ad3c6eaeb5fd1d1698ea4b24a0232aed1ae67bdf4136L4
- **raw-loader** https://github.com/storybookjs/storybook/pull/19368/files#diff-0009efa19beb6daf6560ad3c6eaeb5fd1d1698ea4b24a0232aed1ae67bdf4136L51
- **postcss** and **postcss-loader** https://github.com/storybookjs/storybook/pull/19368/files#diff-0009efa19beb6daf6560ad3c6eaeb5fd1d1698ea4b24a0232aed1ae67bdf4136L59
- **sass-loader** https://github.com/storybookjs/storybook/pull/19368/files#diff-0009efa19beb6daf6560ad3c6eaeb5fd1d1698ea4b24a0232aed1ae67bdf4136L66
- **nanoid** and **p-limit** I don't know. They are not referenced

All of these are not guaranteed to be correct but I don't know how to find out other than looking for references in code files.

**Review by ndelangen — APPROVED** [review]

File: code/frameworks/angular/package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "@storybook/core-common": "7.0.0-alpha.38",
     "@storybook/core-events": "7.0.0-alpha.38",
     "@storybook/core-server": "7.0.0-alpha.38",
+    "@storybook/core-webpack": "7.0.0-alpha.35",
```
Comment (kroeder) [inline code/frameworks/angular/package.json:45]: Sho
```suggestion
    "@storybook/core-webpack": "7.0.0-alpha.38",
```

**Pull request merged**